engine/src/test/java/com/redhat/cloud/notifications/processors/camel/CamelTypeProcessorTest.java

```diff
@@ -76,6 +87,48 @@ class CamelTypeProcessorTest {
     public static final String SUB_TYPE_KEY = "subType";
     public static final String SUB_TYPE = "sub-type";
 
+    /**
+     * The following constants are hard coded in {@link CamelTypeProcessor#callOpenBridge(JsonObject, UUID, String, CamelProperties, String, String)}.
+     */
+    private static final String FIXTURE_TYPE = "myType";
+    private static final String FIXTURE_SOURCE = "notifications";
+    private static final String FIXTURE_SPEC_VERSION = "1.0";
```

Comment:
Let's make them `public static final` fields in `CamelTypeProcessor` and use them from here.

Reply:
This is done too!